### DISCUSSION TITLE
Mouse hover issue on menu bar when remotely controlling Mac (ipad->mac)

### DISCUSSION THREAD

**Discussion by ZhuJiaxin2** [rustdesk/rustdesk#8788] [General]
### Bug Description

When controlling Mac through my iPad, i use apple magic mouse to click on iPad. 
However, when my mouse hover over the menu bar area, the hover function could not work (e.g. my mouse clicks on the apple icon in the menu bar, and several selections were listed under, when my mouse hover downward to reach for the selections, the selections disapears), this issue does not appear in the middle of the screen, when my mouse right clicks and hovers for paste or copy selections.
Also, this has caused great inconvenience, for i could not remotely close my full screen window in Mac! (because of the hovering issue, when my mouse is up there, the close tab button does not appear!)
So grateful for your effort!

### How to Reproduce

I dunno

### Expected Behavior

in Mac, when my mouse click on a menu bar button, and then hover downward for option selection, the selections stay there instead of disappearing under the mouse.

### Operating system(s) on local side and remote side

iPad->Mac

### RustDesk Version(s) on local side and remote side

1.2.6->1.2.6

### Screenshots

Click on menu bar, selections came out:
<img width="1440" alt="截屏2024-07-23 上午11 30 16" src="https://github.com/user-attachments/assets/1a85f5a5-dbdc-46c2-b12e-cbab2dafdf31">
Expectation when mouse hover downward:
<img width="1440" alt="截屏2024-07-23 上午11 30 38" src="https://github.com/user-attachments/assets/cbba4ded-7d2b-4ee5-9a1d-31df0978936e">
What actually happens:
<img width="1440" alt="截屏2024-07-23 上午11 30 28" src="https://github.com/user-attachments/assets/dde962aa-66e2-4c69-ba51-6e2e90bd5c70">

The hidden close tab button:
<img width="1440" alt="截屏2024-07-23 上午11 40 00" src="https://github.com/user-attachments/assets/916c2cc2-f194-4141-b685-0ea86c0f0e74">
The close tab button only appears when mouse hovers there, but this doesnt work on this case, for the hovering function in that area is disabled
<img width="1440" alt="截屏2024-07-23 上午11 40 05" src="https://github.com/user-attachments/assets/4509ccc4-ff34-4926-bec4-c35a1ed2b894">

**Comment by rustdesk** [comment]
Apologize for the wrong conversion. https://github.com/rustdesk/rustdesk/issues/8789
